README.md

```diff
@@ -190,6 +190,24 @@ Now, the ```Category``` model will keep the counter cache in ```special_count```
 
 If you would like to use this with `counter_culture_fix_counts`, make sure to also provide [the `column_names` configuration](#handling-dynamic-column-names).
 
+### Conditional counter cache shorthand
+
+You may also use a conditional API identical to [ActiveRecord's Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html#conditional-callbacks) to conditionally count columns:
+
+```ruby
+class Product < ActiveRecord::Base
+  belongs_to :category
+  counter_culture :category, column_name: :special_count, if: :special?
```

Comment:
I think we should have an example with `unless` as well.

Reply:
Agreed, just wanted to get alignment around the general approach before exploading out the variations

Reply:
Specifically, symbol, proc, array of symbols, array of procs

Reply:
Yeah, I think so